### PR TITLE
Clear old settings and status files

### PR DESCRIPTION
**Pull request by ankitha-p**
In few customer VMs, on upgrading VMSnapshotLinux extension version, the older status files are copied and are not getting cleared out. Added a utility function to clear out stale files in any given directory, with a given extension and file# limit.

### CODE REVIEW

File: VMBackup/main/Utils/HandlerUtil.py [hunk 1]
```diff
@@ -687,25 +689,32 @@ def is_status_file_exists(self):
             self.log("exception is getting status file" + traceback.format_exc())
             return False
 
+    # Rename all .settings and .status files that do not belong to current seq# with '_' suffix
+    # Clear older files in the respective directories
     def backup_settings_status_file(self, _seq_no):
         self.log("current seq no is " + _seq_no)
+        file_extn_settings = '.settings'
+        file_extn_status = '.status'
+        maxLimitOfFiles = 60
```
[the remark this answers is not in the document]
Comment (ankitha-p) [reply VMBackup/main/Utils/HandlerUtil.py:698]: Discussed offline.

**Review by iamashwani-who — APPROVED** [review]

File: VMBackup/main/Utils/HandlerUtil.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (arisettisanjana) [reply VMBackup/main/Utils/HandlerUtil.py:698]: Could you summarize the discussion points about the limit in a way that's easy for everyone to understand?

**Review by arisettisanjana — APPROVED** [review]
